### PR TITLE
Allow adding multiple login IDs

### PR DESCRIPTION
**Pull request by kiootic**

### CODE REVIEW

File: pkg/auth/handler/loginid/add_test.go [hunk 1]
```diff
@@ -110,7 +112,7 @@ func TestAddLoginIDHandler(t *testing.T) {
 					"code": 400,
 					"info": {
 						"causes": [
-							{ "kind": "General", "message": "login ID key is not allowed", "pointer": "/key" }
+							{ "kind": "General", "message": "login ID key is not allowed", "pointer": "/0/key" }
```
Comment (louischan-oursky) [inline pkg/auth/handler/loginid/add_test.go]: Shouldn't the pointer be `/login_ids/0/key`?

**Comment by kiootic** [comment]
@carmenlau Are you testing using SDK example in Chrome?

**Comment by carmenlau** [comment]
I tried calling the api with http client directly, let me try in SDK too.

**Comment by carmenlau** [comment]
I found I configured my http client incorrectly, the apis work! Thanks @kiootic

**Pull request merged**